R/data.R

```diff
@@ -131,9 +131,17 @@ data_admin_init <- function(path_data, path_user = NULL, quiet = FALSE) {
     ## load the symmetric key from the files we're trying to write!
     dat <- data_pub_load(hash, data_path_request(path_data))
     data_authorise_write(path_data, sym, dat, TRUE, quiet)
+
+    dir.create(data_path_template(path_data))
+    file_copy(cyphr_file("template/README.md"),
+              file.path(data_path_cyphr(path_data), "README.md"))
```

Comment:
This will write to a different dir than the `template_request` and `template_authorise` files. Is that intended? Or should this line have a call to `data_path_template` instead of `data_path_cyphr`?

Reply:
This ends up with the README.md in the .cyphr dir, and the templates in templates/ so we get:

```
.cyphr
  README.md
  test
  keys/
  requests/
  templates/
    authorise
    request
```